app/src/components/TracksyWrapper.tsx

```diff
@@ -39,11 +43,25 @@ export function TracksyWrapper({
         console.debug('New uploaded files:', files)
     }
 
+    const handlDemoButtonClick = async () => {
+        console.debug('Demo data loading...')
```

Comment:
Should we remove debug log ?

Reply:
I don't know what is the best practice for this. Would you rather I delete the log or turn into an info log ?
(We already have debug log, isn't that good practice ?)

Reply:
For me, it is best to remove it to keep the user console clean and avoid potential sensitive information leaks.
If we need it, we could have an environment variable "DEBUG=0/1" to enable console.log/info/debug or not